### PR TITLE
Data_Access_Class

### PR DESCRIPTION
**Pull request by khalidmahmud**

### CODE REVIEW

File: MoneyLove/MenuViewController.m [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    if (tableView == self.menuTableView) {
+        return 3;
```
Comment (TequilarTomsk) [inline MoneyLove/MenuViewController.m]: why you compare tableview here? In which case function returns 1?

File: MoneyLove/MenuViewController.m [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    } else {
+        return 1;
+    }
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    static NSString *simpleTableIdentifier = @"Cell";
```
Comment (TequilarTomsk) [inline MoneyLove/MenuViewController.m]: Please change identifier

File: MoneyLove/MenuViewController.m [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:simpleTableIdentifier];
+    if (cell == nil) {
+        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:simpleTableIdentifier];
+    }
+    if (tableView == self.menuTableView) {
+        cell.backgroundColor = [UIColor grayColor];
+    } 
+    return cell;
```
Comment (TequilarTomsk) [inline MoneyLove/MenuViewController.m]: where did set title for menu cell?

File: MoneyLove/MenuViewController.m [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+-(UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger)section
+{
+    UIView *view = [[UIView alloc] initWithFrame:CGRectMake(0, 0, tableView.frame.size.width, 18)];
```
Comment (nguyenvandungb) [inline MoneyLove/MenuViewController.m]: 1. Should dequeue view for table header
2. Need function return height for header view
3. You can use [tableView rectForHeaderForSection] ... to set frame of header view.

**Pull request merged**